src/ByteSizeLib/ByteSizeTypeConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using ByteSizeLib;
+using System.Globalization;
+
+namespace System.ComponentModel
```

Comment:
I'd rather use the `ByteSizeLib` namespace rather than the `System.ComponentModel`.

Reply:
Done